docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
I'm a little confused on why we would be merging in something that would be classified as this? But, if we were too, it makes sense to have its own category